package.json

```diff
@@ -43,9 +43,11 @@
     "@mui/lab": "5.0.0-alpha.151",
     "@mui/material": "5.14.18",
     "@sentry/react": "7.80.1",
+    "classnames": "2.3.2",
```

Comment:
Do you really need that?

Reply:
Yes, It seems as a dependency `react-accessible-treeview` relates on